ui-tests/tools/com.liferay.ide.ui.kaleo/com.liferay.ide.ui.kaleo.wizard/com.liferay.ide.ui.kaleo.wizard.validation/.gitignore

```diff
@@ -0,0 +1,5 @@
+/screenshots
```

Comment:
Please remove all validation files since we didn't have any validation test for kaleo before